R/get_data.R

```diff
@@ -109,38 +143,36 @@ bcdc_get_data.character <- function(record, format = NULL, resource = NULL, ...)
     id_choice <- resource_df$id[resource_df$name == name_choice]
 
     cat("To directly access this record in the future please use this command:\n")
-    cat(glue::glue("bcdc_get_data('{x}', format = '{format}', resource = '{id_choice}')"),"\n")
-
-  }
-
-  ## Bonk if not using interactively
-  if(length(resource_df$ext[resource_df$ext %in% format]) > 1  && is.null(resource) && !interactive()){
-    stop("The record you are trying to access appears to have more than one resource
-         with a glue::glue{format} extension.", call. = TRUE)
+    cat(glue::glue("bcdc_get_data('{x}', resource = '{id_choice}')"),"\n")
 
   }
 
-  ## Go straight through if there aren't multiple resources but the extension is specified
-  if(length(resource_df$ext[resource_df$ext %in% format]) == 1 && is.null(resource)){
-    file_url <- resource_df$url[resource_df$ext == format]
+  ## fail if not using interactively and haven't specified resource
```

Comment:
As mentioned above I think we could just continue if not interactive the same way we do when it's interactive

Reply:
But only for the case of when there is one resource? I think it should still fail in a non-interactive setting if there is more than 1 resource. Just making sure we are on the same page here.

Reply:
Yes, 100% agree.

Reply:
https://github.com/bcgov/bcdata/pull/67/commits/d3d5f87e240f0dbe68ade5b370dd80747b8bf8a1